tests/framework/PostProcessors/MCSimporterPostProcessor/test_MCSimporter_noExpand.xml

```diff
@@ -0,0 +1,58 @@
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/PostProcessors/MCSimporterPostProcessor.MC_importer_Saphire</name>
```

Comment:
Two things here:
First, MC_importer_Saphire is used for both test
Second, the "tests" file is not included, in this case, these tests are not tested by the regression test system

Reply:
item 1 edited and added tests file